src/freenas/usr/local/bin/truenas-grub.py

```diff
@@ -19,18 +19,22 @@ def get_serial_ports():
     # 22.02-RC.2 use real hostid. We need to be able to boot both of these configurations.
     config = [
         'GRUB_DISTRIBUTOR="TrueNAS Scale"',
+        'GRUB_TIMEOUT=10',
```

Comment:
Is this related, or it is a leak?

Reply:
This is related. In the installer issue we realized that 5 seconds is sometimes not enough to get into serial console.

Reply:
In this case it would be giving us time to pick a boot environment. But really it just needs to be enough time to press an arrow key to stop the timeout.

We do provide a way to configure which BE to boot next through the web UI and cli, so there isn't much of a reason to go into this menu unless your BE is broken so badly that it doesn't boot. Hopefully that would be pretty rare.

These are potentially valuable seconds of downtime in the installed system. On the other hand, it's even worse if you miss the boot menu and need to go through another reboot cycle to try again. Ten seconds does feel pretty high, but not unreasonably so.